redash/query_runner/mysql.py

```diff
@@ -46,9 +46,25 @@ def configuration_schema(cls):
                     'type': 'string',
                     'title': 'Database name'
                 },
-                "port": {
-                    "type": "number"
+                'port': {
+                    'type': 'number'
                 },
+                'use_ssl': {
+                    'type': 'boolean',
+                    'title': 'Use SSL'
+                },
+                'ssl_cacert': {
+                    'type': 'string',
+                    'title': 'CA certificate to verify peer against (SSL)'
```

Comment:
No, but we don't encrypt it on the filesystem either. Do you think it's still safer?